### PR TITLE
Release notes for v1.19.1

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: docs/source/release-history.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@
 Release History
 ***************
 
+v1.19.1 (2022-12-09)
+====================
+
+This release fixes compatibility with Python 2.7, which was inadvertently
```
Comment (mrakitin) [inline docs/source/release-history.rst:8]:
```suggestion
This release fixes compatibility with Python 3.7, which was inadvertently
```

**Review by AbbyGi — APPROVED** [review]

**Review by mrakitin — COMMENTED** [review]
Correction of 2.7 -> 3.7 is needed.

File: docs/source/release-history.rst (hunk 1 above)
Comment (danielballan) [reply docs/source/release-history.rst:8]: Oops.

Comment (danielballan) [reply docs/source/release-history.rst:8]: Fixed in fe6a688

**Pull request merged**